plugins/inputs/mysql/mysql.go

```diff
@@ -575,7 +579,7 @@ func (m *Mysql) gatherGlobalVariables(db *sql.DB, serv string, acc telegraf.Accu
 // When the server is slave, then it returns only one row.
 // If the multi-source replication is set, then everything works differently
 // This code does not work with multi-source replication.
-func (m *Mysql) gatherSlaveStatuses(db *sql.DB, serv string, acc telegraf.Accumulator) error {
+func (m *Mysql) gatherSlaveStatuses(db *sql.DB, serv string, acc telegraf.Accumulator, gatherAllChannels bool) error {
```

Comment:
No need to pass `gatherAllChannels` as it is part of the `Mysql` struct; just use `m.GatherAllSlaveChannels` where needed

Reply:
Removed in the most recent commit